.github/workflows/reusable-build.yml

```diff
@@ -45,6 +54,8 @@ jobs:
       BUILD_CONFIGURATION: ${{matrix.configurations}}
       BUILD_PLATFORM: x64
       BUILD_OPTIONS: ${{inputs.build_options}}
+      CXX_FLAGS: ${{inputs.cxx_flags}}
+      LD_FLAGS: ${{inputs.ld_flags}}
```

Comment:
I'd recommend we just create 2 new configurations for FuzzingDebug and FuzzingRelease

Reply:
#1150  filed to track this. Libfuzzer needs the sanitizer options enabled, so it makes more sense to just build a *Sanitizer configuration.